.github/blunderbuss.yml

```diff
@@ -119,6 +119,7 @@ assign_issues_by:
   - nicain
 - labels:
   - 'api: datalabeling'
+  - 'api: talent'
```

Comment:
Could you move this to a separate entry? Unless we want to assign this to Ivan as well.

Reply:
oh whoops - only saw the python-samples-reviewers at a quick glance